lineapy/instrumentation/tracer.py

```diff
@@ -505,3 +544,6 @@ def slice(self, name: str) -> str:
 
     def get_working_dir(self) -> str:
         return self.tracer_context.session_context.working_directory
+
+    def pop_control_stack(self) -> LineaID:
+        return self.control_flow_tracker.pop_node()
```

Comment:
why not do this for push as well?

Reply:
introduced context manager, removed this